update-nix-hashes.yml

```diff
@@ -0,0 +1,87 @@
+name: Update Nix Hashes
```

Comment:
Let's move this functionality into the file https://github.com/TarasMazepa/stax/blob/main/.github/workflows/release-distribution-trigger.yml

Release distribution trigger already contains similar code to update homebrew, chocolatey and docker releases, so it makes sense to put nix there too.